styles/styles.css

```diff
@@ -1484,7 +1485,11 @@ main > .section-content-body.section-with-sidebar .section-content-body__text p
 }
 
 /* All except desktop */
-@media only screen and (max-width: 1200px) {
+@media only screen and (max-width: 1200px) {  
+  html {
+    scroll-padding-top: var(--nav-height-mobile);
```

Comment:
Feels like we might need to increase the scroll padding top slightly above the height value of the nav, so that there's not a close spacing between the bottom of the sticky header and section/column text (see attached).

![Screenshot 2023-08-28 at 5 27 18 PM](https://github.com/hlxsites/merative2/assets/1815714/6ff42ba0-9b75-4e68-9d2f-99a437608351)

Can you try adding `--spacer-element-03` to both the mobile and desktop?

```suggestion
    scroll-padding-top: calc(var(--nav-height-mobile) + var(--spacer-element-03));
```

![Screenshot 2023-08-28 at 5 26 41 PM](https://github.com/hlxsites/merative2/assets/1815714/9b3cda73-8994-47de-88d3-a158dca5cbc8)